Issue602

kfeuerherm opened the pull request:
Voici. Espérons que je l'aie fait comme il faut :)

maieul commented:
Oui, c'est parfait. Je regarderai plus en détails demain.

kfeuerherm commented:
D’accord! Il paraît que je serai encore chez moi le samedi si vous avez besoin de moi. Le dimanche je suis en route.

> On Nov 10, 2017, at 18:56, Maïeul Rouquette <notifications@github.com> wrote:
> 
> Oui, c'est parfait. Je regarderai plus en détails demain.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/maieul/ledmac/pull/717#issuecomment-343617733>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AK_X--_MkBrXFCptt_p84BD7HSlR6tCUks5s1OKRgaJpZM4QaP4b>.
>

maieul commented on examples/1-verses-doublenumbering.tex:
```diff
@@ -20,29 +20,29 @@
+This file provides an example of typesetting verse with reledmac using a double line numbering system. 
 
-The new edition add one starting line of verse which was not present in the previous one, so we need to add in margin and textsuperscript the numbering of the older edition. In the middle, we split in two lines of verse what was one line of verse in the first edition.
+The new edition adds a starting line of verse not present in the previous one, so we need to make an addition within the margin and textsuperscript the numbering from the older edition. In the centre, we split what was in the first edition one line of verse into two.
```
midlle of the text? centre est géographique non?

maieul commented on reledmac.dtx:
```diff
@@ -2959,57 +2959,57 @@
 % \begin{itemize}
-%  \item \cs{Xlinenumannotationposition}\oarg{s}\marg{v} to change the position in critical footnotes, \meta{s} for the series of footnotes to which it applies (it can be a comma separated list of series). If \meta{s} is empty, it also applies to annotation in the side of the pages, along the line number.
-%  \item \cs{Xlinenumannotationpositionside}\marg{v} to change the position in the side of pages, along the line numbers. 
+%  \item \cs{Xlinenumannotationposition}\oarg{s}\marg{v} changes the position in critical footnotes with \meta{s} for the series of footnotes to which it applies, where \meta{s} may be a comma-separated list of series. Where \meta{s} is empty, the change applies also to annotations at the sides of the page, alongside the line number.
```
when \meta is empty instead of where?

maieul commented on reledmac.dtx:
```diff
@@ -15726,14 +15726,14 @@
 % \begin{macro}{\xabslineref}
 % If the specified label exists, \cs{xabslineref} gives its absolute line number.
-% That is used usually only by some \macpackage\ internal macros.
+% This is generally usually only by \macpackage\ internal macros.
```
"This is generally usually"? there are two adverbs

kfeuerherm commented:
Voir ci-dessous.

> On Nov 11, 2017, at 10:02, Maïeul Rouquette <notifications@github.com> wrote:
> 
> In examples/1-verses-doublenumbering.tex <https://github.com/maieul/ledmac/pull/717#discussion_r150385312>:
> 
> >  
> -The new edition add one starting line of verse which was not present in the previous one, so we need to add in margin and textsuperscript the numbering of the older edition. In the middle, we split in two lines of verse what was one line of verse in the first edition.
> +The new edition adds a starting line of verse not present in the previous one, so we need to make an addition within the margin and textsuperscript the numbering from the older edition. In the centre, we split what was in the first edition one line of verse into two.
> midlle of the text? centre est géographique non?
> 
Oui, mais “middle” invoque une idée de centre en deux directions au lieu de centre du point de vue horizontal
> 
> In reledmac.dtx <https://github.com/maieul/ledmac/pull/717#discussion_r150385391>:
> 
> >  % \begin{itemize}
> -%  \item \cs{Xlinenumannotationposition}\oarg{s}\marg{v} to change the position in critical footnotes, \meta{s} for the series of footnotes to which it applies (it can be a comma separated list of series). If \meta{s} is empty, it also applies to annotation in the side of the pages, along the line number.
> -%  \item \cs{Xlinenumannotationpositionside}\marg{v} to change the position in the side of pages, along the line numbers. 
> +%  \item \cs{Xlinenumannotationposition}\oarg{s}\marg{v} changes the position in critical footnotes with \meta{s} for the series of footnotes to which it applies, where \meta{s} may be a comma-separated list of series. Where \meta{s} is empty, the change applies also to annotations at the sides of the page, alongside the line number.
> when \meta is empty instead of where?
> 
> 
Comme vous voulez; ici “where” a un sens temporal. Plutôt une question d’usage/de dialecte
> 
> In reledmac.dtx <https://github.com/maieul/ledmac/pull/717#discussion_r150385435>:
> 
> > @@ -15726,14 +15726,14 @@
>  %
>  % \begin{macro}{\xabslineref}
>  % If the specified label exists, \cs{xabslineref} gives its absolute line number.
> -% That is used usually only by some \macpackage\ internal macros.
> +% This is generally usually only by \macpackage\ internal macros.
> "This is generally usually"? there are two adverbs
> 
En effet, c’est gauche. “usually” devrait être “used"

maieul commented:
Thanks, I will do the change locally.

kfeuerherm commented:
Seems best, I agree.

Thanks for this excellent package and the opportunity to make a small contribution.

> On Nov 11, 2017, at 10:27, Maïeul Rouquette <notifications@github.com> wrote:
> 
> Thanks, I will do the change locally.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/maieul/ledmac/pull/717#issuecomment-343672657>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AK_X-_hZ2cDFYZWJU2L7LUvo14NT483fks5s1bzrgaJpZM4QaP4b>.
>
